packages/metal-soy/src/Soy.js

```diff
@@ -59,6 +61,11 @@ class Soy extends IncrementalDomRenderer.constructor {
 				data[params[i]] = component[params[i]].bind(component);
 			}
 		}
+
+		if (component.getTemplateData) {
```

Comment:
`isFunction` would indeed be safer. We usually just check for the presence, but I think it'd be best to use `isFunction` everywhere actually.